packages/ember-toucan-form/src/components/toucan-form.ts

```diff
@@ -0,0 +1,39 @@
+import Component from '@glimmer/component';
+
+import TextareaFieldComponent from '../-private/textarea-field';
+
+import type { WithBoundArgs } from '@glint/template';
+import type { HeadlessFormComponentSignature } from 'ember-headless-form/components/headless-form';
+
+// TODO: Should we export this from ember-headless-form and use it here instead? This is probably fine?
+type UserData = object;
```

Comment:
@simonihmig is this okay? Or do we want to export [`UserData`](https://github.com/CrowdStrike/ember-headless-form/blob/a049a6d4e275b49aa2366993aca435ccfdac9b3f/packages/ember-headless-form/src/-private/types.ts#L4)?

Reply:
Yeah, it think it's ok. We could export it, but nothing that should block us here...